Modules/selectmodule.c

```diff
@@ -875,13 +896,20 @@ devpoll_unregister(devpollObject *self, PyObject *o)
     Py_RETURN_NONE;
 }
 
-PyDoc_STRVAR(devpoll_poll_doc,
-"poll( [timeout] ) -> list of (fd, event) 2-tuples\n\n\
```

Comment:
The hint about the structure of the return value is lost. Same for _epoll.poll_.

Reply:
Good catch! I'd addressed this for `poll.poll()` but not `devpoll.poll()` and `epoll.poll()`. Fixed!